src/main/java/com/slack/kaldb/elasticsearchApi/searchRequest/SearchRequestSort.java

```diff
@@ -0,0 +1,15 @@
+package com.slack.kaldb.elasticsearchApi.searchRequest;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public enum SearchRequestSort {
+  ASC,
```

Comment:
just curious - is there ever a case where `ASC` can be issued?

Reply:
The elasticsearch datasource for Grafana supports it [here](https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/elasticsearch/datasource.ts#L496). For our current use case I don't think you can request an `ASC` sort, as it appears to just do the relevant sorting in the browser.